### PR TITLE
Webhooks/connections refactor

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request (PR) cleans up webhook-connections and aligns it with alerts/monitors.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203498072839428

### CODE REVIEW

File: docs/manage/connections-integrations/webhook-connections/jira-cloud.md [hunk 1]
```diff
@@ -14,15 +18,11 @@ You can send JSON payloads from Sumo Logic alerts as an HTTP POST request to c
 To send webhook alerts to Jira Cloud you need to include a Basic Authentication Header with requests. This requires an API token, for reference on creating a token see [API tokens](https://confluence.atlassian.com/cloud/api-tokens-938839638.html) in Atlassian Cloud Support.
 
 1. Generate an API token using your [Atlassian Account](https://id.atlassian.com/manage/api-token).
-1. Using the email address of your Atlassian account and an API token you can generate the Basic Authentication Header. In the following cURL command replace `<me@example.com>` with your email address and `<my-api-token>` with your API token and run it:
-
+1. Using the email address of your Atlassian account and an API token, you can generate the Basic Authentication Header. In the following cURL command replace `<me@example.com>` with your email address and `<my-api-token>` with your API token and run it:
```
Comment (kimsauce) [inline docs/manage/connections-integrations/webhook-connections/jira-cloud.md:21]:
```suggestion
1. Using the email address of your Atlassian account and an API token, you can generate the Basic Authentication Header. In the following cURL command, replace `<me@example.com>` with your email address and `<my-api-token>` with your API token and run it:
```

File: docs/manage/connections-integrations/webhook-connections/jira-service-desk.md [hunk 2]
```diff
@@ -15,18 +19,14 @@ To send webhook alerts to Jira Service Desk you need to include a Basic Authenti
 
 1. Generate an API token using your [Atlassian Account](https://id.atlassian.com/manage/api-tokens).
 1. Using the email address of your Atlassian account and an API token you can generate the Basic Authentication Header. In the following cURL command replace `<me@example.com>` with your email address and `<my-api-token>` with your API token and run it:
```
Comment (JainM6) [inline docs/manage/connections-integrations/webhook-connections/jira-service-desk.md:21]: comma after "API token"

File: docs/manage/connections-integrations/webhook-connections/jira-service-desk.md [hunk 3]
```diff
@@ -39,34 +39,25 @@ You need the **Manage connections** [role capability](/docs/manage/users-roles
 
 1. Go to **Manage Data \> Monitoring \> Connections**.
```
Comment (JainM6) [inline docs/manage/connections-integrations/webhook-connections/jira-service-desk.md:40]:
```suggestion
1. Go to **Manage Data** > **Monitoring** > Connections**.
```
I believe we should just bold UI elements and not the arrows (direction)

File: docs/manage/connections-integrations/webhook-connections/microsoft-teams.md [hunk 4]
```diff
@@ -27,23 +32,17 @@ You need the **Manage connections** [role capability](../../users-roles/roles/
 This section demonstrates how to create a webhook connection from Sumo Logic to Microsoft Teams.
 
 1. In Sumo Logic, go to **Manage Data \> Monitoring \> Connections**.
```
Comment (JainM6) [inline docs/manage/connections-integrations/webhook-connections/microsoft-teams.md:34]:
```suggestion
1. In Sumo Logic, go to **Manage Data** > **Monitoring** > **Connections**.
```

**Review by JainM6 — CHANGES_REQUESTED** [review]

File: docs/manage/connections-integrations/webhook-connections/jira-service-desk.md (hunk 3 above)
Comment (kimsauce) [reply docs/manage/connections-integrations/webhook-connections/jira-service-desk.md:40]: Yup - this is a long-term project -- a leftover from the initial HTML-to-md conversion.

File: docs/manage/connections-integrations/index.md [hunk 5]
```diff
@@ -2,34 +2,24 @@
...
 
 You can also:
-
 * **Search Connections** to search. 
 * **Edit** Connections. 
 * **Delete** Connections. 
-* View the **Details** of Connections. 
-
-To be able to create a connection, the Manage Connection [role capability](../users-roles/roles/role-capabilities.md) must be set. Contact your administrator for assistance.
+* View the **Details** of Connections.
```
Comment (kimsauce) [inline docs/manage/connections-integrations/index.md:21]: Remove "the"

Comment (kimsauce) [inline docs/manage/connections-integrations/index.md:21]:
```suggestion
* View **Details** of Connections.
```

Comment (kimsauce) [inline docs/manage/connections-integrations/index.md:21]:
```suggestion
* View **Details** of Connections.
```

**Review by MariePMcGarry — APPROVED** [review]
lgtm

**Pull request merged**